### DISCUSSION TITLE
HTTP 500 Error for single user on "Export" page

### DISCUSSION THREAD

**Discussion by devoeger** [kimai/kimai#5170] [Questions & Answers]
I have a single user that cannot access the "Export" page.  I tried multiple browsers, multiple PCs but still get the error.  Nothing shows up in the logs.  The user thinks they may have set a date range that it did not like in their previous session.  Is there a way to reset a single user "cache" or "settings" on the server side?

**Comment by kevinpapst** [comment]
Share logs first please from System > Doctor.

**Comment by devoeger** [comment]
Nothing shows up in the logs.

**Reply by kevinpapst** [reply]
Then in your server / docker logs. A 500 should always leave a trace somewhere

**Comment by devoeger** [comment]
Got it working.  Updated the memory limit in php.ini.  Thanks for your help!
